packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -26,8 +27,8 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
   private $config: CCPluginConfig;
   private $webex: WebexSDK;
   private agentConfig: IAgentProfile;
-  private registered = false;
   private httpRequest: HttpRequest;
+  private webSocketManager: WebSocketManager;
```

Comment:
Decided to add an instance of the `WebSocketManager` here as now we will directly call the `initWebSocket` method to establish a connection with the web socket and use that Promise to setup the config (shown below).